src/main/java/net/dv8tion/jda/core/entities/MessageChannel.java

```diff
@@ -2133,14 +2131,13 @@ protected void handleResponse(Response response, Request<Void> request)
     }
 
     /**
-     * Attempts to remove the reaction from a message represented by the specified {@code messageId}
+     * Attempts to remove the UTF8 emoji reaction from a message represented by the specified {@code messageId}
```

Comment:
Note that this can also be used to remove custom emotes with `name:id` not only unicode. Thus the original sentence was correct and this specification is misleading.